command/agent/http_test.go

```diff
@@ -476,16 +476,31 @@ func TestParseWait_InvalidIndex(t *testing.T) {
 func TestParseConsistency(t *testing.T) {
 	ci.Parallel(t)
 	var b structs.QueryOptions
+	var resp *httptest.ResponseRecorder
+
+	testCases := [3]string{"/v1/catalog/nodes?stale", "/v1/catalog/nodes?stale=true", "/v1/catalog/nodes?stale=false"}
+	for _, url := range testCases {
+		req, err := http.NewRequest("GET",
+			url, nil)
+		if err != nil {
+			t.Fatalf("err: %v", err)
+		}
```

Comment:
The surrounding tests are quite old, but lately we've been migrating to github.com/shoenig/test for assertions; 

```
must.NoError(t, err)
```